addons/binding/org.openhab.binding.amazondashbutton/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="amazondashbutton"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Sample Thing Type -->
+    <thing-type id="dashbutton">
+        <label>@text/dashButtonLabel</label>
+        <description>@text/dashButtonDescription</description>
+
+        <channels>
+            <channel id="press" typeId="press"/>
+        </channels>
+        <config-description>
+            <parameter name="macAddress" type="text" pattern="([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})">
+                <label>@text/dashButtonMacAddressLabel</label>
+                <description>@text/dashButtonMacAddressDescription</description>
+            </parameter>
+            <parameter name="packetInterval" type="integer">
+                <label>@text/dashButtonPacketIntervalLabel</label>
+                <description>@text/dashButtonPacketIntervalDescription</description>
+                <default>5000</default>
+            </parameter>
+        </config-description>
+    </thing-type>
+
+    <!-- Sample Channel Type -->
+    <channel-type id="press">
+        <item-type>Switch</item-type>
```

Comment:
The target platform contains ESH 0.9.0.201609022016, but trigger channels seem to be intoduced in a later version, right?